packages/ui/core-components/src/lib/atoms/inputs/dropdown/dropdownOptionStore.js

```diff
@@ -12,7 +12,7 @@ import { z } from 'zod';
 
 const DropdownOptionSchema = z.object({
 	label: z.union([z.string(), z.number()]).optional().nullable(),
-	value: z.union([z.string(), z.number()]).optional().nullable(),
+	value: z.union([z.string(), z.number()]),
```

Comment:
Why is this change necessary?

I think at the very least we want `value` to be `.nullable()` so that a `null` value returned from a query can be used

Reply:
[bf2c3c6](https://github.com/evidence-dev/evidence/pull/2301/commits/bf2c3c618dd3ca460b1d6a7a3ef1800a9fcf0bf8)